FAQ.md

```diff
@@ -23,7 +23,7 @@ this.setState({ editorState: EditorState.createEmpty() })
 
 ## Why are mentions broken after using `convertFromRaw` and throwing an error?
 
-We design the API to accept an Immutable Map for a Mention. After saving your datastructure to the erver and using `convertFromRaw` the mentions in there are plain objects. I (Nik) believe this is a law in our design and the mention data should be a plain object. Hint: We might fix this with v2.0.0 f the mentions plugin.
+We design the API to accept an Immutable Map for a Mention. After saving your data structure to the server and using `convertFromRaw`, the mentions in there are plain objects. I (Nik) believe this is a law in our design and the mention data should be a plain object. Hint: we might fix this with v2.0.0 of the mentions plugin.
```

Comment:
It's a discussion we started some time ago and I can't remember if we made a choice. There was three possibilities:

1. Allow JavaScript objects as well as Immutable objects
2. Allow only JavaScript objects
3. Allow only Immutable objects

I am wondering if allowing JavaScript objects wouldn't create the false assumption that the editor deals with JavaScript objects under the hood. The thing is that DraftJS is pretty tightly coupled to Immutable and it's great to keep that in mind or you'll likely come across issues that wouldn't make sense otherwise.